__test__/pet.test.js

```diff
@@ -0,0 +1,96 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
```

Comment:
Theres a few things that need fixing in this function:

- Some lines have semi-colons at the ends and some don't, these should all be made consistent,
- The indentation for the body of the `it` blocks only has 2 spaces when everywhere else has 4,
- It would be more readable if there was a blank line between each `it` block to help visually separate everything